packages/datagateway-common/package.json

```diff
@@ -75,7 +78,7 @@
     "prepare": "yarn tsc",
     "test:watch": "react-scripts test --env=jsdom --watch",
     "lint:js": "eslint --ext=tsx --ext=ts --ext=js --ext=jsx --fix ./src",
-    "tsc": "tsc -p tsconfig.lib.json",
+    "tsc": "tsc --watch -p tsconfig.lib.json",
```

Comment:
I probably committed this accidentally. Adding a note here to remember to take it out when this is all over